src/gui/keymap.h

```diff
@@ -54,6 +54,7 @@ class KeyMap {
         NO_MODEL = -1,
         K55,
         K63,
+        K63_WL,
```

Comment:
https://github.com/ckb-next/ckb-next/blob/e2d81007067df6de11d9b0eb99a72f928551b53d/src/gui/keymap.h#L51-L52

Please add it right before _MODEL_MAX.

Reply:
Done.